tests/integration_tests/security/test_custom_seccomp.py

```diff
@@ -137,49 +137,43 @@ def test_working_filter(test_microvm_with_api):
 )
 def test_failing_filter(test_microvm_with_ssh, vm_config_file):
     """Test --seccomp-filter, denying some needed syscalls."""
-    test_microvm = test_microvm_with_ssh
+    microvm = test_microvm_with_ssh
 
     # Configure VM from JSON. Otherwise, the test will error because
     # the process will be killed before configuring the API socket.
     _config_file_setup(test_microvm_with_ssh, vm_config_file)
 
-    _custom_filter_setup(test_microvm, """{
+    _custom_filter_setup(microvm, """{
```

Comment:
if we'd like to be extra sure that this works on every Firecracker thread, we could somehow parametrize this test to run for each Firecracker thread, and making sure that it is killed by SIGSYS on each of these cases.
Do you think it's worth it?